extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -489,7 +538,7 @@ export class AmpAutocomplete extends AMP.BaseElement {
     if (
       opt_input.length < this.minChars_ ||
       !sourceData ||
-      !sourceData.length
+      !(sourceData.length || hasOwn(sourceData, 'html'))
```

Comment:
This is a bit subtle. Can we add a comment or (even better) change the SSR code path to not go through a function named `filterDataAndRenderResults_` if it won't ever actually "filter data"?

Reply:
Good point, added `autocomplete_` as an intermediary method that passes to `filterDataAndRenderResults_` only when relevant.